### ISSUE TITLE
while load video from url, an error happened. 

### ISSUE THREAD

**Issue by 805056790** [bytedeco/javacv#744]
``` 
   @Test
    public void testFromUrl() throws Exception {
        URL url = new URL("http://devdiag685.blob.core.chinacloudapi.cn/pousheng-dev/2017/07/13/bbd44b56-c12b-4d8a-9706-62be41b8c60a.mp4");
        URLConnection urlConnection = url.openConnection();
        InputStream inputStream = urlConnection.getInputStream();

        FFmpegFrameGrabber grabber = new FFmpegFrameGrabber(inputStream);
        grabber.setFormat("mp4");
        grabber.start();

        int length = grabber.getLengthInFrames();
        length = length > 100 ? 100 : length;
        int i = 0;
        Frame frame=null;
        while (i < length) {
            frame = grabber.grab();
            if ((i > 1) && (frame.image != null)) {
                break;
            }
            i++;
        }
        Java2DFrameConverter converter = new Java2DFrameConverter();
        BufferedImage bi = converter.getBufferedImage(frame);
        File targetFile = new File(LOCAL_JPG);
        ImageIO.write(bi, "jpg",targetFile);
        bi.flush();
        inputStream.close();
    }
```

```
org.bytedeco.javacv.FrameGrabber$Exception: avformat_open_input() error -2: Could not open input "java.io.BufferedInputStream@4e50c791". (Has setFormat() been called?)

	at org.bytedeco.javacv.FFmpegFrameGrabber.startUnsafe(FFmpegFrameGrabber.java:535)
	at org.bytedeco.javacv.FFmpegFrameGrabber.start(FFmpegFrameGrabber.java:465)
	at pers.panxin.test.common.media.VideoTest.testFromUrl(VideoTest.java:100)
```

**Comment by 805056790** [comment]
```
FFmpegFrameGrabber grabber = new FFmpegFrameGrabber("/Users/user/Downloads/my_video.mp4");
```

but this would pass.

**Comment by 805056790** [comment]
http://android.techjaffa.xyz/tag/url-into-ffmpegframegrabber-load-video/

there's the same question. Do I want to know is anyone solved this?

**Comment by saudet** [comment]
This has probably been fixed with pull #703. Please try again with JavaCV 1.3.3-SNAPSHOT.
